src/network/networkd-network.h

```diff
@@ -92,6 +92,8 @@ struct Network {
         struct ether_addr *mac;
         uint32_t mtu;
         uint32_t group;
+        uint32_t gso_max_size;
```

Comment:
`config_parse_iec_size()` takes `size_t`. Please use `size_t` for `gso_max_size`.